catalog-info.yaml

```
@@ -1,18 +1,76 @@
 apiVersion: backstage.io/v1alpha1
-manifestVersion: 0.1.1
+kind: System
+metadata:
+  name:  boleto-api
+  title: boleto-api
+  namespace: mundipagg | pagarme
+  description: Gateway de registro online de transações do meio de pagamento de boleto.
+  links:
+    - url: https://mundipagg.atlassian.net/wiki/spaces/PROC/pages/5650448827/Boleto+-+Overview
+      title: Documentação técnica
+      icon: dashboard
+  annotations:
+    github.com/project-slug: mundipagg/boleto-api
+    pagar.me/ci: azure-devops
+    pagar.me/cd: azure-devops
+    stone.co/sox: "true"
+    stone.co/pii: "true"
+spec:
+  type: backend
+  system: mundipagg/boleto-api
+  owner: pagarme/processing-banks
+  lifecycle: production
+---
+apiVersion: backstage.io/v1alpha1
 kind: Component
 metadata:
   name: boleto-api
-  description: Gateway de registro online de transações do meio de pagamento de boleto
-  compliance:
-    gui: false
-    sox: sox
-    pci: none
+  namespace: mundipagg | pagarme
+  title: Documentação técnica
```

Acho que esse title pode ser outro, o que acha?
```suggestion
  title: boleto-api-server
```